inst/@sym/fourier.m

```diff
@@ -17,28 +17,55 @@
 %% If not, see <http://www.gnu.org/licenses/>.
 
 %% -*- texinfo -*-
-%% @deftypefn  {Function File} {@var{FF} =} fourier (@var{f}, @var{x}, @var{k})
+%% @documentencoding UTF-8
+%% @deftypefn  {Function File} {@var{FF} =} fourier (@var{f}, @var{x}, @var{w})
+%% @deftypefnx {Function File} {@var{FF} =} fourier (@var{f})
+%% @deftypefnx {Function File} {@var{FF} =} fourier (@var{f}, @var{w})
 %% Symbolic Fourier transform.
 %%
-%% FIXME: doc, examples
+%% FIXME: doc
 %%
-%% @seealso{laplace}
+%% Examples:
+%% @example
+%% syms x w
+%% f = exp(-2*abs(x))
+%% fourier(f)
+%% fourier(f, w)
+%% fourier(f, x, w)
+%% @end example
+%%
+%% @seealso{ifourier,laplace}
 %% @end deftypefn
 
 %% Author: Colin B. Macdonald
 %% Keywords: symbolic
 
-function F = fourier(f, x, k)
+function F = fourier(varargin)
+
+  syms f x w;
+  if (nargin == 1)
+    f=varargin{1};
+    x=symvar(f,1);
```

Comment:
I prefer `x = symvar(f, 1)` (just differing whitespace), probably not a big deal but would match the rest of the code